hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/action/Action.java

```diff
@@ -59,9 +47,6 @@
     @JsonInclude(Include.NON_NULL)
     private Event event;
 
-    @JsonInclude(Include.NON_EMPTY)
-    private Map<String, String> properties;
-
     @JsonInclude(Include.NON_NULL)
     private String result;
```

Comment:
It may be captured elsewhere but do we persist other "result" information like actionStartTime, actionEndTime, etc?